docs/FAQ.md

```diff
@@ -1,5 +1,44 @@
 ---
 title: ❓FAQ
+description: Your hottest questions about SourceCred, answered.
 ---
 
-TODO
+### Isn't SourceCred gameable? How do you prevent Sybil attacks?
+
+The Sybil resistance comes from a mix of transparency and community moderation. Cred updates retroactively and payouts are long term, so if you get attacked by Sybils (i.e. duplicate accounts farming cred) or an exploit/bug that gives someone too much cred, it's easy to see, and you have time to respond via moderation.
```

Comment:
Perhaps a good opportunity to use some language I've seen us use before. Something along the lines of. "In short, yes. SourceCred's philosophy is that all systems are gamable. The goal is not to eliminate gaming, but to be resistant to it. SourceCred's Sybil resistance..."